sagemaker-training-compiler/tensorflow/multiple_gpu_single_node/vision-transformer.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<p>delete the commented out code.</p><p> </p><p>Move the note about SageMaker metric resolution to the markdown under Analysis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>